grafana/dashboards/iml-metrics.json

```diff
@@ -2122,7 +2137,7 @@
           "tags": []
         },
         {
-          "alias": "Write Bytes",
+          "alias": "Write Bytes $tag_target",
```

Comment:
```suggestion
          "alias": "Write Bytes $tag_fs",
```

I think we should group by fs at the top level for R/W bytes and R/W iops.

This will change the view from 

![Screen Shot 2020-06-02 at 3 43 14 PM](https://user-images.githubusercontent.com/458717/83562787-25b1d200-a4e8-11ea-8121-2de9f5d9a6e2.png)

to

![Screen Shot 2020-06-02 at 3 43 57 PM](https://user-images.githubusercontent.com/458717/83562801-2b0f1c80-a4e8-11ea-9e34-5f4ff712557d.png)

I think this view will become too busy when there are many targets, and it also seems like we should measure throughput at the fs level instead of all targets (even though that's kind of strange as well).